oteapi_dlite/strategies/oceanlab_influx_parser.py

```diff
@@ -0,0 +1,253 @@
+"""Strategy for oceanlab data parsing from Influx DB."""
+
+import sys
+from typing import Annotated, Optional
+
+import cachetools  # type: ignore
+import dlite
+import influxdb_client
+import jinja2
+from oteapi.models import AttrDict, HostlessAnyUrl, ParserConfig, ResourceConfig
+from pydantic import Field
+from pydantic.dataclasses import dataclass
+
+from oteapi_dlite.models import DLiteSessionUpdate
+from oteapi_dlite.utils import get_collection, update_collection
+from oteapi_dlite.utils.utils import get_meta
+
+if sys.version_info >= (3, 10):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
+
+
+class InfluxParseParseConfig(AttrDict):
+    """Configuration for DLite Excel parser."""
+
+    id: Annotated[
+        Optional[str], Field(description="Optional id on new instance.")
+    ] = None
+
+    label: Annotated[
+        Optional[str],
+        Field(
+            description="Optional label for new instance in collection.",
+        ),
+    ] = "json-data"
+
+    resourceType: Optional[Literal["resource/url"]] = Field(
+        "resource/url",
+        description=ResourceConfig.model_fields["resourceType"].description,
+    )
+    downloadUrl: Optional[HostlessAnyUrl] = Field(
+        None,
+        description=ResourceConfig.model_fields["downloadUrl"].description,
+    )
+    mediaType: Optional[str] = Field(
+        None,
+        description=ResourceConfig.model_fields["mediaType"].description,
+    )
+    storage_path: Annotated[
+        Optional[str],
+        Field(
+            description="Path to metadata storage",
+        ),
+    ] = None
+    collection_id: Annotated[
+        Optional[str], Field(description="A reference to a DLite collection.")
+    ] = None
+
+    url: Annotated[Optional[str], Field(description="url to the db")] = None
+
+    USER: Annotated[Optional[str], Field(description="user to the db")] = None
+
+    PASSWORD: Annotated[
+        Optional[str], Field(description="user pwd to the db")
+    ] = None
+
+    DATABASE: Annotated[Optional[str], Field(description="database name")] = (
+        None
+    )
+
+    RETPOLICY: Annotated[
+        Optional[str], Field(description="retpolicy for db")
+    ] = None
+
+
+class InfluxParseStrategyConfig(ParserConfig):
+    """DLite excel parse strategy  config."""
+
+    configuration: Annotated[
+        InfluxParseParseConfig,
+        Field(description="DLite json parse strategy-specific configuration."),
+    ]
+
+
+class InfluxParseSessionUpdate(DLiteSessionUpdate):
```

Comment:
So... we're not doing SessionUpdate stuff anymore. While it may not be necessary to update this throughout the code base, it should definitely be updated in this file, I think, no?